table/format.cc

```diff
@@ -403,4 +404,17 @@ Status UncompressBlockContents(const UncompressionInfo& uncompression_info,
                                                    ioptions, allocator);
 }
 
+Status SanitizeDbHostIdProperty(Env* env, std::string& db_host_id) {
+  if (db_host_id == kDbHostId) {
+    db_host_id.clear();
+    db_host_id.resize(kMaxHostNameLen);
+    Status s = env->GetHostName(&db_host_id.at(0), kMaxHostNameLen);
+    if (!s.ok()) {
+      return s;
```

Comment:
Doesn't this leave the db_host_id non-empty?

Reply:
Good point.